pkg/apis/core/v1/conversion_test.go

```diff
@@ -36,7 +36,7 @@ import (
 	"k8s.io/kubernetes/pkg/api/legacyscheme"
 	"k8s.io/kubernetes/pkg/apis/core"
 	corefuzzer "k8s.io/kubernetes/pkg/apis/core/fuzzer"
-	corev1 "k8s.io/kubernetes/pkg/apis/core/v1"
+	k8s_api_v1 "k8s.io/kubernetes/pkg/apis/core/v1"
```

Comment:
underscored aliases read strangely to me... I prefer the existing aliases as-is

Reply:
Hi @liggitt , thanks for your attention. 
the aliases in kubernetes are so chaotic, I think we can have some standard, not casual.

Reply:
`corev1` maps to `.../core/v1` better than `k8s_api_v1`

Reply:
essentially, the algorithm is if the package name (in this case, `v1`) is ambiguous, include the next package level up as well (`corev1`)

Reply:
+1 to what @liggitt wrote - we are trying to use the convention of using the shortest suffx (in terms of number of elements of the path) that makes it unique. So corev1 here is what we want.